lib/gds_api/test_helpers/rummager.rb

```diff
@@ -6,47 +6,76 @@ module TestHelpers
     module Rummager
       RUMMAGER_ENDPOINT = Plek.current.find('rummager')
 
-      def stub_any_rummager_post
-        stub_request(:post, %r{#{RUMMAGER_ENDPOINT}/documents})
+      def stub_any_rummager_post(index: nil)
+        if index
+          stub_request(:post, %r{#{RUMMAGER_ENDPOINT}/#{index}/documents})
+            .to_return(status: [202, "Accepted"])
+        else
+          stub_request(:post, %r{#{RUMMAGER_ENDPOINT}/documents})
+            .to_return(status: [202, "Accepted"])
+        end
       end
 
       def stub_any_rummager_post_with_queueing_enabled
+        warn "stub_any_rummager_post_with_queueing_enabled is deprecated: use stub_any_rummager_post instead"
+
         stub_request(:post, %r{#{RUMMAGER_ENDPOINT}/documents}) \
           .to_return(status: [202, "Accepted"])
       end
 
-      def assert_rummager_posted_item(attributes)
-        url = RUMMAGER_ENDPOINT + "/documents"
-        assert_requested(:post, url) do |req|
+      def assert_rummager_posted_item(attributes, index: nil, **options)
+        if index
+          url = RUMMAGER_ENDPOINT + "/#{index}/documents"
+        else
+          url = RUMMAGER_ENDPOINT + "/documents"
+        end
```

Comment:
What sort of thing would you pass as an `index` argument? When would you explicitly set an index vs. posting to `/documents` ?